README.md

```diff
@@ -22,6 +22,7 @@ Older proposals were managed in documents. All new proposals should be submitted
 | [Report Aggregation](https://github.com/kyverno/KDP/pull/32) | Implemented | 1.8 |
 | [Store Kyverno policies in OCI registries](https://docs.google.com/document/d/15cqD4HPecI5Uv2u1Yfg0JCgWDVi2HLwGZPvTX_48W2E/edit?usp=sharing) | Implemented | 1.9 |
 | [Policy Exceptions](https://github.com/kyverno/KDP/pull/33) | In Review | 1.9 |
+| [Cache enhancement with Informers](/proposals/cache_enhancements.md) | Implemented | 1.9 |
```

Comment:
Since the KDP was for ConfigMap, maybe call it "ConfigMap cache" or similar?

Reply:
fixed